.github/workflows/build.yml

```diff
@@ -16,9 +16,27 @@ on:
     - 3.7
 
 jobs:
+  check_source:
+    name: 'Check for source changes'
+    runs-on: ubuntu-latest
+    outputs:
+      run_tests: ${{ steps.check.outputs.run_tests }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check for source changes
+        id: check
+        run: |
+          if [ -z "GITHUB_BASE_REF" ]; then
```

Comment:
Typo here, it should be
```bash
if [ -z "$GITHUB_BASE_REF" ]; then
```

The if never triggers because we are checking if the string literal `"GITHUB_BASE_REF"` is empty, it is not. The `$GITHUB_BASE_REF` value should be there.